Gopkg.toml

```diff
@@ -0,0 +1,41 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+[[constraint]]
+  name = "github.com/spf13/cobra"
+  branch = "master"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/HotelsDotCom/flyte"
+
+[[override]]
+  branch = "good-citizen"
```

Comment:
Hopefully it will be one day. the way go-logger runs it's init is a bit tricky with other packages. It is quite invasive and even known I'm not using it but because I'm using flyte as a dependency it is brining transitive dependencies which change the way other packages are interacting with go's logger. The branch I'm referring to is my suggestion to how go-logger should work. I don't expect it to be merged anytime soon

Reply:
We'll need a reminder to update this to `master` when the `good-citizen` branch in go-logger is merged. Deleting merged branches is customary in GitHub and will cause problems if this update is forgotten.